src/main/java/hattrickdata/Arena.java

```diff
@@ -0,0 +1,31 @@
+package hattrickdata;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+
+import java.util.Optional;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@EqualsAndHashCode
+@ToString
```

Comment:
Probably a bit late to the party, but I am not super keen on adding a dependency to lombok in HO.  Could this be turned into Java records, or better still use Kotlin?  We have been trying to move to Kotlin in recent times, and the language would naturally offer those facilities.